core/modules/system/system.install

```diff
@@ -3038,6 +3038,17 @@ function system_update_1067() {
   $config->save();
 }
 
+/**
+ * Convert the file_scan_ignore_directories variable to use configuration.
+ */
+function system_update_1068() {
+  $config = config('system.core');
+  $config->set('file_scan_ignore_directories', update_variable_get('file_scan_ignore_directories', array()));
```

Comment:
...not sure where to put this, so I went with `system.install`. Please let me know if you feel that it belongs elsewhere.